packages/get-starknet/src/rpcs/supported-wallet-api.ts

```diff
@@ -0,0 +1,14 @@
+import type { RpcTypeToMessageMap } from 'get-starknet-core';
+
+import { SupportedWalletApi } from '../constants';
+import { StarknetWalletRpc } from '../utils/rpc';
+
+export type WalletSupportedWalletApiMethod = 'wallet_supportedWalletApi';
+type Params = RpcTypeToMessageMap[WalletSupportedWalletApiMethod]['params'];
+type Result = RpcTypeToMessageMap[WalletSupportedWalletApiMethod]['result'];
+
+export class WalletSupportedWalletApi extends StarknetWalletRpc {
+  async handleRequest(_param: Params): Promise<Result> {
+    return SupportedWalletApi;
```

Comment:
i think the response format is not correct?

it is
``` 
   [{
      api_version: "some version"
   }]
```

Reply:
Based on what I read here : https://github.com/starkware-libs/starknet-specs/blob/6d88b7399f56260ece3821c71f9ce53ec55f830b/wallet-api/wallet_rpc.json#L19 It seems to be an array of strings not array of object.

Reply:
we can use string[] for now , but await for clarify